<?php

use Symfony\Component\DependencyInjection\Argument\RewindableGenerator;
use Symfony\Component\DependencyInjection\Exception\RuntimeException;

// This file has been auto-generated by the Symfony Dependency Injection Component for internal use.
// Returns the public 'scheb_two_factor.firewall_context' shared service.

include_once $this->targetDirs[3].'\\vendor\\scheb\\two-factor-bundle\\Security\\TwoFactor\\TwoFactorFirewallContext.php';
include_once $this->targetDirs[3].'\\vendor\\scheb\\two-factor-bundle\\Security\\TwoFactor\\TwoFactorFirewallConfig.php';

return $this->services['scheb_two_factor.firewall_context'] = new \Scheb\TwoFactorBundle\Security\TwoFactor\TwoFactorFirewallContext(['contao_backend' => new \Scheb\TwoFactorBundle\Security\TwoFactor\TwoFactorFirewallConfig(['auth_form_path' => 'contao_backend_login', 'check_path' => 'contao_backend_two_factor', 'auth_code_parameter_name' => 'verify', 'always_use_default_target_path' => false, 'default_target_path' => '/', 'success_handler' => NULL, 'failure_handler' => NULL, 'authentication_required_handler' => NULL, 'trusted_parameter_name' => '_trusted', 'multi_factor' => false, 'csrf_token_generator' => NULL, 'csrf_parameter' => '_csrf_token', 'csrf_token_id' => 'two_factor', 'provider' => NULL])]);
